de/django_templates/README.md

```diff
@@ -103,4 +103,4 @@ Glückwunsch! Nun kannst du dich daran machen, neue Posts in deinem Django Admin
 
 Funktioniert super? Wir sind so stolz auf dich! Steh kurz auf und geh ein Stück weg vom Computer. Du hast dir eine Pause verdient :)
 
-![Abbildung 13.4](images/donut.png)
\ No newline at end of file
+![Abbildung 13.4](images/donut.png)
```

Comment:
Looks like there wasn't a newline at the end of line before, so the GitHub web editor added one? Let me know if I should remove it again, but I suppose others might want to edit the tutorial via the GitHub editor as well.

Reply:
I'm in favor of newlines a the end so it's fine.